src/g_game.c

```diff
@@ -1653,10 +1653,12 @@ static void G_WriteLevelStat(void)
         }
     }
 
-    fprintf(levelstat, "%s%s - %s (%s)  K: %d/%d  I: %d/%d  S: %d/%d\n",
+    fprintf(fstream, "%s%s - %s (%s)  K: %d/%d  I: %d/%d  S: %d/%d\n",
             levelString, (secretexit ? "s" : ""),
             levelTimeString, totalTimeString, playerKills, totalkills,
             playerItems, totalitems, playerSecrets, totalsecret);
+
+    fclose(fstream);
```

Comment:
So, we didn't even close the `levelstat.txt` file handler until now? I wonder why this bug wasn't spotted earlier.

Reply:
I think the Windows standard C library (CRT) has been updated and changed its behavior. I can't replicate it on my local Win10 build, but our release build is latest Windows Server with all updates installed and has static CRT, so release artifacts have this bug.

Reply:
Appending after each level is better anyway, thank you!